app/controllers/organizations/staff_controller.rb

```diff
@@ -1,6 +1,9 @@
 class Organizations::StaffController < Organizations::BaseController
   layout "dashboard"
 
+  before_action :organization_admin
+
   def index
+    @staff_accounts = StaffAccount.where(organization: current_tenant)
```

Comment:
```suggestion
    @staff_accounts = StaffAccount.all
```
This shouldn't need to be scoped since ActsAsTenant auto scopes things for us.